pandas/plotting/_core.py

```diff
@@ -1213,8 +1213,12 @@ def hist(self, by=None, bins=10, **kwargs):
         ----------
         by : str or sequence, optional
             Column in the DataFrame to group by.
-        bins : int, default 10
+        bins : str or int, default 10
```

Comment:
```suggestion
        bins : int or str, default 10
```

the default typically goes first.